src/com/csciOOD/BouncingObject.java

```diff
@@ -1,56 +1,44 @@
 package com.csciOOD;
-import com.csciOOD.Game;
-import java.awt.*;
-
-public class BouncingObject extends Game {
-    private float x = 0;
-    private float y = 0;
-    private float lastX = 0;
-    private float lastY = 0;
-    private int width = 42;
-    private int height = 42;
-    private int speed = 20;
-    private int xVelocity = (int) Math.random() * speed;
-    private int yVelocity = (int) Math.random() * speed;
+
+public class BouncingObject {
+    public float x = 200;
+    public float y = 200;
+    public int width = 42;
+    public int height = 42;
+    public float speed = 2;
+    public float xVelocity = 2;
+    public float yVelocity = 2;
+
+    public BouncingObject(){}
 
     private boolean wallWasHit(){
-        boolean leftWallHit = x + width / 2 >= getWidth();
+        boolean leftWallHit = x + width / 2 >= 800;
```

Comment:
Dimensions are hard-coded all over the place for brevity, but it would be nice to do this in a more flexible, single source of truth kind of way.